src/components/store/shopItems/shopItems.selectors.ts

```diff
@@ -0,0 +1,7 @@
+import { GetShopItemsResponseData } from './shopItems.actions';
+
+type State = {
```

Comment:
Use you RootState type here

https://react-redux.js.org/using-react-redux/usage-with-typescript